### ISSUE TITLE
Aria adjustment, related to submenu behavior

### ISSUE THREAD

**Issue by jack-robs** [OperationCode/front-end#665]
# Feature request
`Aria` needs some tweaking

## Proposed solution
Currently you have `aria-expanded`, `aria-haspopup` on the links to main pages, such as _About_. But this isn't true. When you use a screenreader and tab to the link, your screenreader will indicate that it has a popup, so the user expects that when they hit <kbd>enter</kbd>, a menu will pop up, when in reality pressing <kbd>Enter</kbd> when you're on _About_ will just take you to the _About_ page.

The button that triggers the submenu should have `aria-haspopup`. The button also needs a label so a screenreader user knows what it does. I attached a gif showing how Voiceover reads it currently. It just indicates it's a button that is not or is expanded, but there is no way to tell it expanded a submenu for _About_.

![recording of navigating the Operation Code menu with Voiceover on](https://i.imgur.com/iPNgwwX.gif)

This menu example is what you want to aim for.
https://codepen.io/aardrian/pen/JaadKE

The button is labeled and associated with it's page using `aria-labeledby`.

## Additional context
a11y audit

**Comment by kylemh** [comment]
Resolved in #667
